server_test.go

```diff
@@ -111,7 +111,7 @@ func genTestAuthCookie() *http.Cookie {
 
 	cookie := &http.Cookie{}
 	cookie.Name = AuthCookieName
-	cookie.Value = base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("%s%s%s", string(rawBody), "::hmac::", string(mac.Sum(nil)))))
+	cookie.Value = padBase64(base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("%s%s%s", string(rawBody), "::hmac::", base64.StdEncoding.EncodeToString((mac.Sum(nil)))))))
```

Comment:
I forgot to run `go test` after making an update to how we encode the cookie, and this test cookie slipped by causing tests to fail. While we should prolly pull this out into a function to be more DRY with the code, it is likely not that big of a deal yet. For sure, if it comes up a third time, it will need a small refactoring.